config/Make.rules.Linux

```diff
@@ -186,35 +164,15 @@ mkshlibname     = lib$(1).so
 # Clear the iconv ldflags, iconv is part of libc on Linux
 iconv_ldflags   :=
 
-#
-# With GCC < 6.1, libbacktrace crash with a SEGV for -pie exes linked with debug-stripped libs
-# See https://gcc.gnu.org/ml/gcc-patches/2016-03/msg00132.html
-# Here we assume only release builds occasionally get -g and later get stripped
-# TODO: enable libbacktrace for release builds with GCC >= 6.1
-ifneq ($(OPTIMIZE),yes)
-libbacktrace_fullpath := $(shell $(CXX) --print-file-name=libbacktrace.a)
-ifneq ($(libbacktrace_fullpath),libbacktrace.a)
-   libbacktrace = yes
-endif
-endif
-
-IceUtil_system_libs                             = -lrt $(if $(filter yes,$(libbacktrace)),-lbacktrace)
-Ice_system_libs                                 = -ldl -lssl -lcrypto $(IceUtil_system_libs)
-ifeq ($(shell pkg-config --exists libsystemd 2> /dev/null && echo yes),yes)
-Ice_system_libs                                 += $(shell pkg-config --libs libsystemd)
-endif
+IceUtil_system_libs                             = -lrt -lbacktrace
+Ice_system_libs                                 = -ldl -lssl -lcrypto $(IceUtil_system_libs) $(shell pkg-config --libs libsystemd)
```

Comment:
what happens if there's no systemd, does `$(shell pkg-config --libs libsystemd)` return empty?

Reply:
Do you think we should keep the check for libsystemd installed, or just emit an error if missing?

Reply:
Keep the check. For instance it's not going to be available in Docker builds.

Reply:
Fixed as suggested

Reply:
Thanks!